core/src/main/java/org/elasticsearch/index/analysis/CustomAnalyzer.java

```diff
@@ -63,10 +69,24 @@ public TokenizerFactory tokenizerFactory() {
         return tokenizerFactory;
     }
 
+    /**
+     * Names of the token filters.
```

Comment:
These are needed by the analysis API but not really anywhere else. They aren't a huge burden to maintain though, not like the `name()` member was.